src/data.tsx

```diff
@@ -1647,10 +1931,10 @@ export const quests: Quest[] = [
         ],
       },
       {
+        name: "Satellite Master Part 2",
+        requires: "Satellite Master Part 1",
         description:
           "Visit Starport Landing Pad Uplink. Visit Skeleton Observation Uplink.",
-      },
-      {
```

Comment:
Did I have an extra part in this quest or was the merging of these parts accidental?